### ISSUE TITLE
Issues with BGP with mixed quad graphs

### ISSUE THREAD

**Issue by rubensworks** [joachimvh/SPARQLAlgebra.js#44]
If quad patterns with different graphs are present in a BGP, then only a single graph will be picked for the `GRAPH` clause when SPARQLifying, while each of them should be wrapped in a different `GRAPH`.

Algebra:
```json
{
 "type": "project",
 "input": {
  "type": "bgp",
  "patterns": [
   {
    "subject": {
     "value": "b1"
    },
    "predicate": {
     "value": "http://example.org/hero"
    },
    "object": {
     "value": "hero"
    },
    "graph": {
     "value": ""
    },
    "type": "pattern"
   },
   {
    "subject": {
     "value": "hero"
    },
    "predicate": {
     "value": "http://example.org/name"
    },
    "object": {
     "value": "hero_name"
    },
    "graph": {
     "value": "hero_graph"
    },
    "type": "pattern"
   }
  ]
 },
 "variables": [
  {
   "value": "hero_graph"
  },
  {
   "value": "hero_name"
  }
 ]
}
```

SPARQL actual:
```sparql
SELECT ?hero_graph ?hero_name WHERE {
  GRAPH ?hero_graph {
    ?b2 <http://example.org/hero> ?hero.
    ?hero <http://example.org/name> ?hero_name.
  }
}
```

SPARQL expected:
```sparql
SELECT ?hero_graph ?hero_name WHERE {
  ?b2 <http://example.org/hero> ?hero.
  GRAPH ?hero_graph {
    ?hero <http://example.org/name> ?hero_name.
  }
}
```

**Comment by joachimvh** [comment]
Actually more annoying than expected :D
Fixed in ad560dffecdb6a2f0cc0e7cecf077a232d3d4420

New version (1.5.1) has been published.

**Comment by rubensworks** [comment]
Thanks!
